src/useForm.ts

```diff
@@ -92,6 +92,7 @@ export function useForm<
   context,
   defaultValues = {} as DefaultValues<TFieldValues>,
   shouldFocusError = true,
+  shouldUseCustomValidity = false,
```

Comment:
```suggestion
  shouldUseNativeValidation = false,
```

Shouldn't be `shouldUseNativeValidation `?